app/_src/gateway-operator/get-started/konnect/deploy-data-plane.md

```diff
@@ -8,22 +8,29 @@ chapter: 2
 To attach a {{ site.kgo_product_name }} data plane to {{ site.konnect_short_name }} the data plane needs to know which endpoint to connect to, and how to authenticate the requests.
 
 To get the endpoint and the authentication details of the data plane.
-1.  [Log in to {{ site.konnect_short_name }}](https://cloud.konghq.com/login).
-1.  Navigate to {% konnect_icon runtimes %} [**Gateway Manager**](https://cloud.konghq.com/us/gateway-manager), choose the control plane, and click **Create a New Data Plane Node**.
-1.  In the **Create a Data Plane Node** page select *Kubernetes* as the **Platform**.
-1.  Create a namespace named `kong` in the Kubernetes cluster
-    ```bash
-    kubectl create namespace kong
-    ```
-1.  Click **Generate Certificate**  in step 3.
+
+1. [Log in to {{ site.konnect_short_name }}](https://cloud.konghq.com/login).
+1. Navigate to {% konnect_icon runtimes %} [**Gateway Manager**](https://cloud.konghq.com/us/gateway-manager), choose the control plane, and click **New DataPlane Node**.
+1. In the **Create a Data Plane Node** page select *Kubernetes* as the **Platform** and choose assign it a name. Do not complete the steps therein defined, as we'll use a different procedure.
```

Comment:
```suggestion
1. In the **Create a Data Plane Node** page select *Kubernetes* as the **Platform** and assign it a name. 
_**Do not** complete the steps defined in {{site.konnect_short_name}}, as we'll use a different procedure._
```

Reply:
Actually, the note about not completing the process was a leftover. I've removed it altogether.